.idea/runConfigurations.xml

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@tervay is this not being caught by the gitignore?

Reply:
No, it provides some default buttons for intellij users like deploy. i put it in there intentionally. @sammyredha undelete this line please

Reply:
I have...no idea what that line is, never seen it

The filepath is weird, too - `runConfigurations` is a folder for me, not a file.